school_library_decorator/teacher.rb

```diff
@@ -1,12 +1,14 @@
-require './person.rb'
+require './person'
+# rubocop:disable Style/OptionalBooleanParameter
```

Comment:
- [ ] Please don't disable linter rules. Disabling rubocop might help with linter errors but these are here to help us write clean code.